src/app/return-credentials/return-credentials.component.ts

```diff
@@ -10,35 +11,42 @@ export class ReturnCredentialsComponent implements OnInit {
 
   navLinks: any[];
   activeLinkIndex = -1;
+  private clientType: string;
 
   ngOnInit() {
     this.router.events.subscribe((res) => {
       this.activeLinkIndex = this.navLinks.indexOf(this.navLinks.find(tab => tab.link === '.' + this.router.url));
   });
   }
 
-  constructor(private router: Router) {
-    this.navLinks = [
-      {
-        label: 'Income Tax',
-        link: './incomeTax',
-        index: 0
-      },
-      {
-        label: 'TDS',
-        link: './tds',
-        index: 1
-      },
-      {
-        label: 'GST',
-        link: './gst',
-        index: 2
-      },
-      {
-        label: 'ROC',
-        link: './roc',
-        index: 3
-      },];
-  }
-
-}
+  constructor(private router: Router, private dataTransferService: DataTransferService) {
+      this.dataTransferService.currentClientType.subscribe(clientType => this.clientType = clientType);
+        this.navLinks =  [
+          {
+            label: 'Income Tax',
+            link: './incomeTax',
+            index: 0
+          },
+          {
+            label: 'TDS',
+            link: './tds',
+            index: 1
+          },
+          {
+            label: 'GST',
+            link: './gst',
+            index: 2
+          },
+          {
+            label: 'ROC',
+            link: './roc',
+            index: 3
+          }
+         ];
+      }
+      checkClientType(label: any): boolean {
+            if((label.toString()=="ROC") && ((this.clientType=="company") || (this.clientType=="default")))
```

Comment:
What is clientType== default? Where is the condition to disable ROC tab when clientype is company or LLP?

Reply:
was a typo corrected